pkg/streamingpromql/planning/core/selectors.go

```diff
@@ -44,6 +44,12 @@ func describeSelector(matchers []*LabelMatcher, ts *time.Time, offset time.Durat
 		builder.WriteString(offset.String())
 	}
 
+	if anchored {
+		builder.WriteString(" anchored")
+	} else if smooothed {
+		builder.WriteString(" smoothed")
+	}
```

Comment:
If for some reason both were set on the selector, I would want that to be visible:

```suggestion
	if anchored {
		builder.WriteString(" anchored")
	}
	if smooothed {
		builder.WriteString(" smoothed")
	}
```